### ISSUE TITLE
Decrease informational logs

### ISSUE THREAD

**Issue by magic7s** [grzegorz914/homebridge-meraki-control#11]
Would it be possible to change [this](https://github.com/grzegorz914/homebridge-meraki-control/blob/master/index.js#L346) log to a debug level? It prints out ever 10 seconds and fills up the log quickly providing only static information I entered in the configuration. Or move it to print only on the initial plugin startup. 

```
[27/09/2021, 11:32:46] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:32:46] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:32:46] [Meraki] Network: Meraki Network
[27/09/2021, 11:32:46] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:32:46] [Meraki] Organization Id: 5555555
[27/09/2021, 11:32:46] [Meraki] ----------------------------------
[27/09/2021, 11:32:56] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:32:56] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:32:56] [Meraki] Network: Meraki Network
[27/09/2021, 11:32:56] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:32:56] [Meraki] Organization Id: 5555555
[27/09/2021, 11:32:56] [Meraki] ----------------------------------
[27/09/2021, 11:33:04] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:33:04] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:33:04] [Meraki] Network: Meraki Network
[27/09/2021, 11:33:04] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:33:04] [Meraki] Organization Id: 5555555
[27/09/2021, 11:33:04] [Meraki] ----------------------------------
[27/09/2021, 11:33:04] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:33:04] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:33:04] [Meraki] Network: Meraki Network
[27/09/2021, 11:33:04] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:33:04] [Meraki] Organization Id: 5555555
[27/09/2021, 11:33:04] [Meraki] ----------------------------------
[27/09/2021, 11:33:04] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:33:04] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:33:04] [Meraki] Network: Meraki Network
[27/09/2021, 11:33:04] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:33:04] [Meraki] Organization Id: 5555555
[27/09/2021, 11:33:04] [Meraki] ----------------------------------
[27/09/2021, 11:33:07] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:33:07] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:33:07] [Meraki] Network: Meraki Network
[27/09/2021, 11:33:07] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:33:07] [Meraki] Organization Id: 5555555
[27/09/2021, 11:33:07] [Meraki] ----------------------------------
[27/09/2021, 11:33:17] [Meraki] -------- Meraki Network --------
[27/09/2021, 11:33:17] [Meraki] Manufacturer: Cisco/Meraki
[27/09/2021, 11:33:17] [Meraki] Network: Meraki Network
[27/09/2021, 11:33:17] [Meraki] Network Id: L_59999999999999999
[27/09/2021, 11:33:17] [Meraki] Organization Id: 5555555
[27/09/2021, 11:33:17] [Meraki] ----------------------------------
```

**Comment by grzegorz914** [comment]
The log print out only one time after HB restart or after connections lose to Meraki server. 
I don’t know why You get this every 10sec.

**Comment by grzegorz914** [comment]
Probably fixed with 0.10.6
